### PR TITLE
Change limelight/shooting values

### PR DESCRIPTION
**Pull request by JosephCarpenter**

### CODE REVIEW

**Review by AndrewLester — APPROVED** [review]

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -227,17 +237,30 @@ private void configureButtonBindings() {
         driveSubsystem.setDefaultCommand(new RunCommand(
             () -> {
                 if (robot.isOperatorControlEnabled()) {
-                    driveSubsystem.joystickDrive(-leftJoystick.getY() * 0.7, rightJoystick.getX() * 0.7);
+                    driveSubsystem.joystickDrive(-leftJoystick.getY()*0.7, rightJoystick.getX()*0.7);
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/RobotContainer.java]: Why no spaces?

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -227,17 +237,30 @@ private void configureButtonBindings() {
...
+        // btnLauncherSolenoid.whenHeld(
+        //     new AutomaticShootCommand(TARGET_VELOCITY, -1, shooterSubsystem).perpetually(),
+        //     true
+        // );
+
+        btnLauncherSolenoid.whenPressed(new InstantCommand(() -> shooterSubsystem.activatePiston()));
+        btnLauncherSolenoid.whenReleased(new InstantCommand(() -> shooterSubsystem.lowerPiston()));
+
+        btnLauncherMotor.whenPressed(new InstantCommand(() -> {
+            if (limelight.getPlaneDistance()>-1)
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/RobotContainer.java]: A space here would also be helpful for readability

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 3]
```diff
@@ -107,4 +107,8 @@ public boolean isAtTargetSpeed() {
         double currentRPM = this.shooterEncoder.getVelocity();
         return (Math.abs(currentRPM - targetRPM) <= 100);
     }
+
+    public int getDistanceToRPM(int distance) {
+        return (int)((distance*(-1.05469))+4432.2);
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/ShooterSubsystem.java:112]: Don't use magic numbers. Where did these come from?!

Comment (JosephCarpenter) [reply src/main/java/frc/robot/subsystems/ShooterSubsystem.java:112]: We ran the shooter at different distances to make a linear equation of the needed RPM

Comment (ArchdukeTim) [reply src/main/java/frc/robot/subsystems/ShooterSubsystem.java:112]: Maybe variables `speedCoefficient` and `speedOffset`

**Pull request merged**